stock-tracker/src/features/topPeers/redux/actionTypes.ts

```diff
@@ -0,0 +1,8 @@
+export const ADD_TOP_PEERS = "ADD_TOP_PEERS";
+export type ADD_TOP_PEERS = typeof ADD_TOP_PEERS;
+
+export const SET_LOADING_PEERS = "SET_LOADING_PEERS";
+export type SET_LOADING_PEERS = typeof SET_LOADING_PEERS;
```

Comment:
no need to add extra type here, when you create constant, it automatically knew it.
It saves many lines od code